How usage affects job priority within an instance (#1872)
Qiskit/documentation: docs/guides/instances.mdx

Those using the Open Plan instance have up to 10 minutes total of quantum time per month, which resets at 00:00:00 UTC on the first of each calendar month. Open Plan users can track their usage on the [Platform dashboard](https://quantum.ibm.com/), [Jobs](https://quantum.ibm.com/jobs), and [Account](https://quantum.ibm.com/account) pages.

## How usage affects job priority within an instance

*This section applies to Premium Plan users only, since their access includes multiple instances.* When determining which jobs to run, the [fair-share scheduler](./fair-share-scheduler) takes into account an instance's usage of the QPU in question. For example, an instance that has already run many jobs on a certain QPU might not have as high a priority on that QPU, when compared to an instance that has very low usage.

> The jobs you run, as well as the jobs run by other collaborators in the same instance, count toward the reported usage for that instance. You can see the usage for a selected instance on your [dashboard](https://quantum.ibm.com/) (use the [instance switcher](#switch-instances) at the top of the page to change which instance is reflected on your dashboard).  

Reviewer: ```suggestion
The jobs you run and the jobs run by other collaborators in the same instance count toward the reported usage for that instance. You can see the usage for an instance on your [dashboard](https://quantum.ibm.com/) (use the [instance switcher](#switch-instances) at the top of the page to change which instance is reflected on your dashboard).  
```

> Be aware that if an instance has been marked "limited" by your administrator and the instance exceeds its allocation (also defined by your administrator) within the 28-day rolling window, jobs run with that instance will not complete until more time is available. If an instance is not limited, and if the instance exceeds its allocation, jobs run with that instance are likely to run at a lower priority and experience longer queue times.

Reviewer: ```suggestion
If an instance has been marked as "limited" by your administrator (you will see a "**Remaining**" column for the usage) and the instance exceeds its allocation (defined by your administrator) within the 28-day rolling window, jobs run with that instance will not complete until more time is available. If an instance is not limited and the instance exceeds its allocation, jobs run with that instance are likely to run at a lower priority and experience longer queue times.
```

> An alert will display on an instance's usage when it has exceeded its allocation.

Reviewer: ```suggestion
An alert will display on an instance's usage when it has exceeded its allocation.
```
```suggestion
An alert displays on an instance's usage when it has exceeded its allocation.
```
